Dockerfile

```diff
@@ -52,5 +52,15 @@ RUN rm -rf /usr/local/tomcat/webapps/*
 COPY server.xml /usr/local/tomcat/conf
 COPY --from=build /src/dhis-2/dhis-web/dhis-web-portal/target/dhis.war /usr/local/tomcat/webapps/ROOT.war
 
-# Expose the easy-to-remember directory /DHIS2_home for Docker volume mounting to configure the CORE instance
+# Create the easy-to-remember directory /DHIS2_home for Docker volume mounting to configure the CORE instance
+RUN mkdir /DHIS2_home
+
+RUN addgroup tomcatusers && \
+    adduser -g tomcatusers tomcat --disabled-password && \
```

Comment:
I think we should use `adduser -S` and `addgroup -S` to create a system user and a system group.

Reply:
I would use `tomcat:tomcat` but that is only my _personal preference_ so that's not part of the review.

Reply:
done; changed the group name to tomcat as well

Reply:
🎉

Reply:
tomcat user looks good.  Instead of 'mkdir /DHIS2_home' why don't you 'mkdir /opt/dhis2' instead?  That way you don't have to set the environment variable as it is the default.